src/app/users/users.service.ts

```diff
@@ -62,7 +64,7 @@ export class UsersService {
         switchMap(userIdentity => this.httpClient.get<UserPermission[]>(
           // TODO: Request permissions for given scope via specific API call, e.g.
           // GET /users/{userId}/permissions/{scope}
```

Comment:
To have the endpoint `GET /users/{userId}/permissions/{scope}` conformed with all the rest endpoints related to outbound permissions I think it would make sense to change the description of the TODO as `TODO: Request permissions for given scope via specific API call, e.g. GET /users/{userId}/outbound-permissions?scope={scope}`.

Here I suggest to pass the scope parameter as a query parameter (not a path variable) since we want to be able to fetch all outbound parameters as well as to fetch filtered outbound parameters by scope. Having the endpoint `GET /users/{userId}/outbound-permissions/{scope}` we will have to support the second endpoint `GET /users/{userId}/outbound-permissions`. With a query parameter `scope` we can use the only one endpoint `GET /users/{userId}/outbound-permissions` to serve both filtered outbound permissions as well as all outbound permissions.

The endpoint `GET /users/{userId}/outbound-permissions?scope={scope}` serving outbound permissions is what we have at the moment. Thereby I suppose we can remove the TODO from here.
@svetivanova what do you think?

Reply:
I totally agree with you, so i removed TODO here too